test/e2e/framework/framework.go

```diff
@@ -26,15 +30,24 @@ type Framework struct {
 
 var Global *Framework
 
+var (
+	domain        = zms.DomainName("k8s.omega.stage.kube-test")
+	roleName      = zms.EntityName("syncer-e2e")
+	roleResource  = zms.ResourceName("syncer-e2e")
+	trustroleName = zms.EntityName("test-trustrole")
+	trustdomain   = "prod-eng.omega.acceptancetest.trust-domain"
+	namespace     = "prod--eng-omega-acceptancetest-test--domain"
```

Comment:
For this namespace, we can convert the test domain command line argument into the corresponding namespace mapping.